resources/roca_main.brs

```diff
@@ -15,7 +15,7 @@ function main() as object
 
         if suite = invalid then
             print "Error running tests: Runtime exception occurred in " + filePathWithoutPkg
-            return
+            return {}
```

Comment:
I suppose `return invalid` would be enough to silence that error, too — I'm fine with either though so take your preference!

Reply:
I like the empty object just because it feels a little safer.